source/js/components/pulse-project-list/pulse-project-list.jsx

```diff
@@ -29,7 +29,7 @@ export default class PulseProjectList extends React.Component {
       });
     });
 
-    projectXHR.open(`GET`, `https://${this.props.env.PULSE_API_DOMAIN}/api/pulse/v2/entries/?format=json&search=${query}${this.props.featured ? `&featured=True` : ``}`);
+    projectXHR.open(`GET`, `https://${this.props.env.PULSE_API_DOMAIN}/api/pulse/v2/entries/?format=json&page_size=${this.props.max ? this.props.max : 48}&search=${query}${this.props.featured ? `&featured=True` : ``}`);
```

Comment:
So 0 gets evaluated as false in the ternary and thus fetches 48?

Reply:
Correctamundo.